src/Autocomplete/composer.json

```diff
@@ -26,6 +26,7 @@
     "require": {
         "php": ">=8.1",
         "symfony/dependency-injection": "^6.3|^7.0",
+        "symfony/deprecation-contracts": "^2.5|^3",
         "symfony/http-foundation": "^6.3|^7.0",
         "symfony/http-kernel": "^6.3|^7.0",
         "symfony/property-access": "^6.3|^7.0",
```

Comment:
Below, in `require-dev`, we should allow for `doctrine/orm` 2 AND 3 now, so both are tested.